<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1926</width>
    <height>1460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_connect_device">
    <property name="geometry">
     <rect>
      <x>654</x>
      <y>825</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/mysource/picture/button_background.png);
font: 20pt &quot;AcadEref&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>连接设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_play_groups">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>820</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/mysourece/picture/button_background.png);
font: 20pt &quot;AcadEref&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>上场组别</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_start_game">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>820</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/mysourece/picture/button_background.png);
font: 20pt &quot;AcadEref&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>开始比赛</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_rules">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>900</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/mysourece/picture/button_background.png);
font: 20pt &quot;AcadEref&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>比赛规则</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_software_note">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>900</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/mysourece/picture/button_background.png);
font: 20pt &quot;AcadEref&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>历史积分</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_close">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>900</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>退出比赛</string>
    </property>
   </widget>
   <zorder>label_logo</zorder>
   <zorder>button_play_groups</zorder>
   <zorder>button_rules</zorder>
   <zorder>button_connect_device</zorder>
   <zorder>button_start_game</zorder>
   <zorder>button_software_note</zorder>
   <zorder>button_close</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1926</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
